src/test/resources/examples/resultset/valid-resultset-example.json

```diff
@@ -39,14 +39,24 @@
   "processFolders": [
     {
       "name": "salmon",
-      "path": "./",
+      "path": "./salmon",
       "children": [
         {
           "name": "salmon.merged.gene_tpm",
           "fileType": "tsv",
-          "path": "salmon/salmon.merged.gene_tpm"
+          "path": "./salmon/salmon.merged.gene_tpm"
         }
       ]
     }
-  ]
+  ],
+  "runID": {
```

Comment:
Regarding the `runID`, that is given by Tower, do you recall if we get returned this ID from Tower when calling the API, @KevinMenden ?
I'm just trying to think how we would programmatically get this one.

Reply:
Hmm I don't really remember how that worked, but I think you might even have to know that to access the API anyway :thinking: so we would have to store it somewhere anyway I guess.

Reply:
if the API follows to HATEOAS principle, than the HTTP response should return a URL to this resource (containing the run id).

Reply:
Yes, exactly, the API response returns the Run ID, tested it yesterday :)